talk/owt/sdk/base/peerconnectiondependencyfactory.cc

```diff
@@ -265,63 +260,36 @@ PeerConnectionDependencyFactory::CreatePeerConnectionOnCurrentThread(
   std::unique_ptr<cricket::PortAllocator> port_allocator;
   port_allocator.reset(new cricket::BasicPortAllocator(
       network_manager_.get(), packet_socket_factory_.get()));
-  // Handle port ranges.
-  IcePortRanges ice_port_ranges;
-  GlobalConfiguration::GetPortRanges(ice_port_ranges);
-  if (ice_port_ranges.audio.min > 0 &&
-      ice_port_ranges.audio.max > ice_port_ranges.audio.min) {
-    port_allocator->SetAudioPortRange(ice_port_ranges.audio.min,
-                                      ice_port_ranges.audio.max);
-  }
-  if (ice_port_ranges.video.min > 0 &&
-      ice_port_ranges.video.max > ice_port_ranges.video.min) {
-    port_allocator->SetVideoPortRange(ice_port_ranges.video.min,
-                                      ice_port_ranges.video.max);
-  }
-  if (ice_port_ranges.screen.min > 0 &&
-      ice_port_ranges.screen.max > ice_port_ranges.screen.min) {
-    port_allocator->SetScreenPortRange(ice_port_ranges.screen.min,
-                                       ice_port_ranges.screen.max);
-  }
-  if (ice_port_ranges.data.min > 0 &&
-      ice_port_ranges.data.max > ice_port_ranges.data.min) {
-    port_allocator->SetDataPortRange(ice_port_ranges.data.min,
-                                     ice_port_ranges.data.max);
-  }
   return (pc_factory_->CreatePeerConnection(config, std::move(port_allocator),
-                                            nullptr, observer))
-      .get();
+                                            nullptr, observer));
```

Comment:
Where do we specify the port ranges?

Reply:
The port range API is still in GlobalConfiguration, but it's not passed to port allocator. Created an issue for tracking https://github.com/open-webrtc-toolkit/owt-client-native/issues/619.